core/include/detray/geometry/volume_graph.hpp

```diff
@@ -66,36 +68,38 @@ class volume_graph {
     struct node_generator
         : public detray::ranges::view_interface<node_generator> {
 
-        /// A node in the graph has an index (volume index) and a collction of
+        /// A functor to retrieve the half-edges of a node
+        struct node_builder {
```

Comment:
Since the surfaces of a volume no longer all reside in the bruteforce search method, they are now  are now visited by a functor for each volume